survival_guide/3_operations.adoc

```diff
@@ -590,32 +590,45 @@ Our recommended configuration for using the Spark History Server with Fusion is
 [source,yaml]
 ----
 gcs:
-    enableGCS: false
-    secret: history-secrets
-    key: [SECRET_KEY_NAME].json
-    logDirectory: gs://[BUCKET_NAME]
+  enableGCS: true
+  secret: history-secrets
+  key: sparkhistory.json
+  logDirectory: gs://[BUCKET_NAME]
 service:
-    type: ClusterIP
-    port: 18080
+  type: ClusterIP
+  port: 18080
 ----
-We override the default `service.type` of `LoadBalancer` with `ClusterIP` to keep the History Server from being accessible to outside connections without port-forwarding.
+The `service` key sets up the history server on an internal IP within your cluster but does not create an external LoadBalancer (which is the default setting in the Spark History Server Helm chart). This prevents the server from being exposed to outside access by default. We’ll look at how to access the history server shortly.
+
+The `key` and `secret` fields provide the Spark History Server with the details of where it will find an account with access to the Google Cloud Storage bucket given in `logDirectory`. In the following example, we're going to set up a new service account that will be shared between the Spark History Server and the Spark driver/executors for both viewing and writing logs.
```

Comment:
Be consistent with what we are calling it on this page after it's introduced - i.e. "Spark History Server" or "history server". use the same term every time with the same capitalization. This applies to the whole page not just this usage.